proxy/handlers/hub.go

```diff
@@ -0,0 +1,128 @@
+package handlers
+
+import (
+	"github.com/ElrondNetwork/notifier-go/filters"
```

Comment:
sort imports

Reply:
remove all empty lines and then run `goimports`

Reply:
done